src/Core/ServerSettings.h

```diff
@@ -71,7 +71,11 @@ namespace DB
     M(String, index_mark_cache_policy, DEFAULT_INDEX_MARK_CACHE_POLICY, "Secondary index mark cache policy name.", 0) \
     M(UInt64, index_mark_cache_size, DEFAULT_INDEX_MARK_CACHE_MAX_SIZE, "Size of cache for secondary index marks. Zero means disabled.", 0) \
     M(Double, index_mark_cache_size_ratio, DEFAULT_INDEX_MARK_CACHE_SIZE_RATIO, "The size of the protected queue in the secondary index mark cache relative to the cache's total size.", 0) \
-    M(UInt64, mmap_cache_size, DEFAULT_MMAP_CACHE_MAX_SIZE, "A cache for mmapped files.", 0) \
+    M(UInt64, mmap_cache_size, DEFAULT_MMAP_CACHE_MAX_SIZE, "Maximum number of files to keep in the mmapped file cache.", 0) \
+    M(String, secondary_index_cache_policy, DEFAULT_SECONDARY_INDEX_CACHE_POLICY, "Index mark cache policy name.", 0) \
```

Comment:
Can there be a use case for the primary and secondary index caches using different eviction policies? Just curious :)

Reply:
¯\_(ツ)_/¯ , I just added the same few settings as caches above. IIRC primary index doesn't have a cache, it's just always kept in memory as long as the data part is loaded. For the other caches above - I guess the only way this would come up is if someone notices (e.g. sees it in profiler) that one of the caches isn't working well and tries different settings for just that cache.